source/github-helpers/index.ts

```diff
@@ -39,8 +39,6 @@ export const getCurrentBranch = (): string | undefined => {
 
 export const isFirefox = navigator.userAgent.includes('Firefox/');
 
-export const getRepoURL = (): string => getRepositoryInfo().url!.toLowerCase();
```

Comment:
I'm tempted to preserve this function so that we have a shortcut, but this would then obfuscate the fact that it calls `getRepositoryInfo` and it might therefore be misused in functions that need both `url` and `path`, for example

Reply:
Agree But I like how short and clean it was.

Reply:
Went with the shortcut, now this isn't too bad:

```diff
- getRepoURL()
+ getRepo()!.url
```

Reply:
Looks good